src/gluonts/dataset/common.py

```diff
@@ -505,6 +505,8 @@ def serialize_field(field):
             field = field.astype(np.object_)
             field[nan_ix] = "NaN"
             return field.tolist()
+        if isinstance(field, (int, float)):
```

Comment:
This needs further investigation. I'm not sure it's doing what it's supposed to do.